reserved_keyword_checker/reserved_keyword_checker.py

```diff
@@ -102,10 +103,14 @@ def local_app(self):
         """
         Return whether or not this violation was found in a local source file or a package
         installed in the Python environment running this code. In this case, 'local' means
-        in the source code files in the directory that this is being run in. This can be determined
-        by using the module_name of the Violation, which is a path to the Python module.
+        in the source code files in the directory that this is being run in. This can be
+        determined by checking if the Django app containing this violation is in env
         """
-        return os.path.isfile(self.module_name)
+        # This will return the path to the virtualenv/python installation being used to
+        # run this django app (i.e. /edx/app/credentials/venvs/credentials)
+        env_base_path = '/'.join(sys.executable.split('/')[:-2])
```

Comment:
Nit: it's good to get into the habit of using the more portable os.path library.
```suggestion
        from os.path import dirname
        env_base_path = dirname(dirname(sys.executable))  # remove the "bin/python" suffix.
```